pycqed/analysis_v3/saving.py

```diff
@@ -164,7 +164,7 @@ def save_fit_results(data_dict, fit_res_dict, **params):
 
 def save_figures(data_dict, figs, **params):
 
-    key_list = params.get('key_list', 'auto')
```

Comment:
logger needs name (could not place comment at top of file because it's not in the diff)